### DISCUSSION TITLE
这难道是龙猫的多语言测试吗

### DISCUSSION THREAD

**Discussion by Chiloven945** [Meloong-Git/PCL#2171] [提问]
![image](https://github.com/Hex-Dragon/PCL2/assets/116699482/eeb0f01a-2253-417a-965f-d5fc21feebd8)
🤔（？

**Comment by MoYuan-CN** [comment]
同问

**Comment by iPodtouch6557** [comment]
0.4.4dev甚至有英文和繁体中文选项……可以翻译部分文字
